libs/wire-api/src/Wire/API/Routes/Public/Galley.hs

```diff
@@ -353,8 +354,9 @@ data Api routes = Api
 
 type ServantAPI = ToServantApi Api
 
+-- | Deprecated. Use FeatureConfigGet.
```

Comment:
it's not deprecated.  it's still useful for team admins who want to distinguish the settings on a user from the overridden settings on the team.

Reply:
Removed it and updated the documenation.